src/session.rs

```diff
@@ -31,18 +35,45 @@ impl SessionWrapper {
             .build()
             .await
             .map_err(err_to_napi)?;
-        Ok(SessionWrapper {
-            internal_session: sb,
-        })
+        Ok(SessionWrapper { internal: sb })
```

Comment:
The variable name should be `s`, not `sb`, because it's an already built Session, not SessionBuilder.